.travis.yml

```diff
@@ -48,6 +48,10 @@ install:
     - pip install -r requirements.txt
     - pip install -r requirements_test.txt
 
+    # image gradients plug-ins
+    - pip install plugins/data/imageGradients
+    - pip install plugins/view/imageGradients
```

Comment:
No wait, yes you do.
```
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-59l1vv-build/setup.py", line 5, in <module>
        from digits.extensions.data import GROUP as DIGITS_PLUGIN_GROUP
    ImportError: No module named digits.extensions.data
```
I think this Travis build is about to fail, let's see ...
https://travis-ci.org/NVIDIA/DIGITS/builds/162241842